portfolio/templates/main/home_page.html

```diff
@@ -7,32 +7,24 @@
 
 {% block content %}
 {% wagtailuserbar %}
+        {% if page.featured_entries %}
```

Comment:
The three loops aren't really an issue. I would make this a little more efficient by using a `with` clause to stash the queryset in a variable. 
```
{% with featured_entries=page.featured_entries %}
    /* all your stuff here using featured_entries instead of page.featured_entries */
{% endwith %}
```